build.rs

```diff
@@ -157,6 +162,19 @@ fn produce_version_db() -> Result<JuliaupVersionDB> {
                         version: format!("{}+0~x86", v),
                     },
                 );
+            } else if target_os == "linux" {
+                db.available_channels.insert(
+                    format!("{}", v),
+                    JuliaupVersionDBChannel {
+                        version: format!("{}+0~x64", v),
+                    },
+                );
+                db.available_channels.insert(
+                    format!("{}~x64", v),
+                    JuliaupVersionDBChannel {
+                        version: format!("{}+0~x64", v),
+                    },
+                );
```

Comment:
This should all be `aarch64`. I think Linux doesn't have emulation, right? If it did, we could add additional platforms here, i.e. that would then add the ability to say run the x64 binaries on aarch64 on Linux, if that was supported.